crates/sui/src/validator_commands.rs

```diff
@@ -557,6 +575,83 @@ impl SuiValidatorCommand {
                     }
                 }
             }
+            SuiValidatorCommand::UpdateBridgeCommitteeNodeUrl {
+                bridge_authority_url,
+                print_unsigned_transaction_only,
+                validator_address,
+            } => {
+                // Make sure the address is member of the committee
+                let address = if !print_unsigned_transaction_only {
+                    let address = context.active_address()?;
+                    if let Some(validator_address) = validator_address {
+                        if validator_address != address {
+                            bail!(
+                                "`--validator-address` must be the same as the current active address: {}",
+                                address
+                            );
+                        }
+                    }
+                    address
+                } else {
+                    validator_address
+                        .ok_or_else(|| anyhow!("--validator-address must be provided when `print_unsigned_transaction_only` is true"))?
+                };
+                let sui_rpc_url = &context.config.get_active_env().unwrap().rpc;
+                let bridge_client = SuiBridgeClient::new(sui_rpc_url).await?;
+                let committee_members = bridge_client
+                    .get_bridge_summary()
+                    .await
+                    .map_err(|e| anyhow!("{e:?}"))?
+                    .committee
+                    .members;
+                if !committee_members
+                    .into_iter()
+                    .any(|(_, m)| m.sui_address == address)
+                {
+                    bail!("Address {} is not in the committee", address);
+                }
+                println!(
+                    "Updating bridge committee node URL for Sui validator: {address}, url: {}",
+                    bridge_authority_url
+                );
+
+                let bridge = bridge_client
+                    .get_mutable_bridge_object_arg_must_succeed()
+                    .await;
+
+                let gas = context
```

Comment:
would it make sense to check that the gas object has enough balance for the call?
or maybe call `gas_for_owner_budget`? or have the ability to pass a given gas object?